manager/utils.go

```diff
@@ -179,7 +178,12 @@ func GenerateEventName(probeType, funcName, UID string, attachPID int) (string,
 
 	if len(eventName) > MaxEventNameLen {
 		// truncate the function name and UID name to reduce the length of the event
-		eventName = safeEventRegexp.ReplaceAllString(fmt.Sprintf("%s_%s_%s_%d", probeType, funcName[0:int(math.Min(10, float64(len(funcName))))], UID[0:int(math.Min(10, float64(len(UID))))], attachPID), "_")
+		attachPIDstr := strconv.Itoa.attachPID
+		len := (MaxEventNameLen - 4 /* _ and \0 */ - len(probeType) - len(attachPIDstr)) / 2
```

Comment:
Are you we need to include the null character in this length check? It wasn't before.

Reply:
One problem with this math is that it evenly divides the remaining length between the function name and the UID. Often the UID is not specified, and thus it would limit the function name more than it needed to.

Reply:
I simplified the whole thing, I think the real question is what the minimal function name length to validate the event name

Reply:
I think preserving the `UID` is most important, because it is used to cleanup old entries upon `Manager` start. The function name part of the event string is just convenience, so it can be truncated to ensure we meet the length requirements. @Gui774ume does this sound correct?

Reply:
Hey 👋🏻 
Sorry for the late answer, I missed the notification.
Yes both the `UID` and the `Pid` are required for the cleanup to work (the first one is used to make sure we are deleting only the probes of an instance of the current manager, and the second one to check that we are cleaning up probes only from dead managers). Looking at Nicolas' last changes, afaik it looks good to me !